client/src/app.jsx

```diff
@@ -48,20 +61,128 @@ class App extends React.Component {
     });
   }
 
-  // getBusinessAddress() {
-
-  // }
+  getBusinessAddress(ID) {
+    $.ajax({
+      type: 'GET',
+      url: `/businesses/${ID}/business_map`,
+      contentType: 'application/json',
+      success: (data) => {
+        const address = data[0].address;
+        this.setState({
+          address: address,
+          phonenumber: data[0].phone_number,
+        });
+        if (address.includes("Fillmore")) {
+          this.setState({
+            relativeaddress: "b/t Webster St & Steiner St",
```

Comment:
Feel like these information (relativeaddress and relativedistrict) can be stored in database. So you don't have to do the if else for all of them here.

Reply:
on you app.jsx when you 
          if (key === 'mon' || key === 'tue' || key === 'wed' || key === 'thu' || key === 'fri' || key === 'sat' || key === 'sun') {
            businessHours[key] = value;
          }

i would do  something like 
businessHours[key] = {val: value, tag: key}
that way you have both the values and tag that you can just toUpperCase later